### PR TITLE
change to use operator for listing targets when possible

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

**Comment by aviramha** [comment]
Currently fails deserialization?

File: Cargo.toml [hunk 1]
```diff
@@ -54,7 +54,7 @@ futures = "0.3"
 thiserror = "1"
 k8s-openapi = { version = "0.20", features = ["v1_24"] }
 reqwest = { version = "0.11", default-features = false, features = ["blocking", "rustls-tls-native-roots", "json", "socks"] }
-kube = { git = "https://github.com/metalbear-co/kube", default-features = false, features = ["runtime", "derive", "client", "ws", "rustls-tls", "oidc", "socks5", "http_proxy"] }
+kube = { git = "https://github.com/metalbear-co/kube", default-features = false, features = ["runtime", "derive", "client", "ws", "rustls-tls", "oidc", "socks5", "http_proxy"], branch = "fix_mb" }
```
Comment (DmitryDodzin) [inline Cargo.toml:57]: maybe merge `fix_mb` first?

**Review by DmitryDodzin — APPROVED** [review]
Small Nitpick =]

File: Cargo.toml (hunk 1 above)
Comment (aviramha) [reply Cargo.toml:57]: I actually prefer to have it branched to avoid pushing by error

**Pull request merged**